wordpress/cloud-config.yaml

```
@@ -0,0 +1,91 @@
+## template:glesys
+#cloud-config
+# Licensed under the CC0 1.0
+
+{{>users}}
+
+packages:
+    - apache2
+    - php7.4-fpm
+    - curl
+    - php7.4-curl
+    - php7.4-mysql
+    - php7.4-gd
+    - certbot
+    - python3-certbot-apache
+    - mysql-server
+    - fail2ban
+write_files:
+    -
+        content: |
+            <VirtualHost *:80>
+            ServerName {{params.hostname}}
+            ServerAlias {{#params.hostnames}}{{name}} {{/params.hostnames}}
+
+            DocumentRoot /var/www/html
+                    <Directory /var/www/html>
+                            Options -Indexes +FollowSymLinks +MultiViews
+                            AllowOverride All
+                            Require all granted
+
+                            <files xmlrpc.php>
+                                Require all denied
+                            </files>
+
+                            #PHP-FPM Socket
+                            <FilesMatch \.php$>
+                            SetHandler "proxy:unix:/var/run/wordpress.sock|fcgi://localhost/"
```

Indenteringen är ett off här.

```suggestion
                                SetHandler "proxy:unix:/var/run/wordpress.sock|fcgi://localhost/"
```